src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -24,18 +57,174 @@ const formConfig = {
+                'ui:required': (formData) => !!formData && (formData.onBehalfOf !== 'I want to submit my complaint anonymously')
+              },
+              middle: {
+                'ui:title': 'Your middle name'
+              },
+              suffix: {
+                'ui:title': 'Your suffix'
+              },
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: isNotAnonymous
+              }
+            }),
+            dob: {
+              'ui:title': 'Date of birth',
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: isNotAnonymous
+              }
+            },
+            serviceAffiliation: { // could wrap service info in an object
+              'ui:title': 'Service affiliation',
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: 'Myself'
+              },
+              'ui:required': (formData) => !!formData && (formData.onBehalfOf === 'Myself')
+            },
+            serviceBranch: {
+              'ui:title': 'Branch',
+              'ui:options': {
+                hideIf: (formData) => {
+                  if (!formData.serviceAffiliation || formData.serviceAffiliation !== 'Veteran') {
+                    return true;
+                  }
+                  return false;
+                },
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: 'Myself'
+              }
+            },
+            serviceDateRange: _.merge(dateRangeUI(
+              'Service start date',
+              'Service end date',
+              'End of service must be after start of service'
+            ),
+            {
+              'ui:options': {
+                hideIf: (formData) => {
+                  if (!formData.serviceAffiliation || formData.serviceAffiliation !== 'Veteran') {
+                    return true;
+                  }
+                  return false;
+                },
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: 'Myself'
+              }
+            }),
+            email: {
+              'ui:title': 'Email',
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: 'I want to submit my complaint anonymously'
+              }
+            }
           },
           schema: {
             type: 'object',
+            required: [
+              'onBehalfOf',
+            ],
             properties: {
-              name: {
+              onBehalfOf: {
+                type: 'string',
+                'enum': [
+                  'Myself',
+                  'Someone else',
+                  'I want to submit my complaint anonymously'
+                ]
+              },
+              fullName: {
+                type: 'object',
+                properties: {
+                  // prefix: {
```

Comment:
Let's go ahead and enable this, and include prefix in the uiSchema definition above as:
  ```
prefix: {
    'ui:title': 'Prefix',
    'ui:options': {
      widgetClassNames: 'form-select-medium'
    }
  }
```

Reply:
Okay, so are we implementing this in design as well? Just trying to make some continuity checks..